src/main/java/teammates/common/datatransfer/AdminEmailAttributes.java

```diff
@@ -153,4 +154,20 @@ public String getCreateDateForDisplay(){
     public String getContentForDisplay(){
         return StringHelper.recoverFromSanitizedText(this.getContent().getValue());
     }
+    
+    public String getFirstAddressReceiver(){
+        return this.addressReceiver.get(0);
+    }
+    
+    public boolean getHasGroupReceiver() {
+        return this.getGroupReceiver() != null && this.getGroupReceiver().size() > 0;
```

Comment:
Don't know why I thought it was an attribute. Do we really need this function? We can just use `not empty groupReceiver` in JSTL, right?